app/views/api_docs/vendor_api_docs/reference/draft.html.erb

```diff
@@ -0,0 +1,170 @@
+  </li>
+</ol>
+
+<hr class="govuk-section-break govuk-section-break--visible govuk-!-margin-top-6 govuk-!-margin-bottom-6">
+
+<h2 class="govuk-heading-l" id="versions">API Versions</h2>
+
+<p class="govuk-body">
+  The current version of this API is 1.0. The next version of this API will be 1.1.  This is a draft specification for version 1.1.
+</p>
+
+<p class="govuk-body">
+  From version 1.1 onwards, version numbers will follow this format: MAJOR.MINOR (e.g. 1.2)
+</p>
+
+<p class="govuk-body">
+  MAJOR version: The version used in the URI and defines breaking changes to the API.
+</p>
+
+<p class="govuk-body">
+  MINOR version: Includes non-breaking changes to the API, mostly additions. For example, a minor version may add a new field or a new endpoint.
+</p>
+
+<p class="govuk-body">
+  All changes, minor or major, will be documented in our release notes.
+</p>
+
+<p class="govuk-body">
+Minor version updates will become available automatically at the MAJOR version urls. For example, version 1.1 urls will be available automatically under /api/v1 when version 1.1 is released. Pre-existing urls may return additional fields at this point.
+
+<p class="govuk-body">
+We will also support requesting a specific minor version via the url, see examples of this within the <%= govuk_link_to 'Developing on the API', '#developing', class: 'app-contents-list__link' %> section.
+</p>
+
+<h2 class="govuk-heading-l" id="important">Version 1.1 changes</h2>
+
+<p class="govuk-heading-s">Interviews</p>
+<p class="govuk-body">
+  We now support creating, updating and cancelling interviews.
+</p>
+
+<p class="govuk-heading-s">Notes</p>
+<p class="govuk-body">
+  We now support creating notes.
+</p>
+
+<p class="govuk-heading-s">Deferring applications</p>
+<p class="govuk-body">
+  We now support deferring an offer to the next cycle.
+</p>
+
+<p class="govuk-heading-s">Withdrawing applications</p>
+<p class="govuk-body">
+  We now support withdrawing an application at the candidate's request.
+</p>
+
+<p class="govuk-heading-s">Pagination</p>
+<p class="govuk-body">
+  Pagination has been added to the API through a page parameter as well as a per_page parameter.
+</p>
+
+<hr class="govuk-section-break govuk-section-break--visible govuk-!-margin-top-6 govuk-!-margin-bottom-6">
+
+<h2 class="govuk-heading-l" id="developing">Developing on the API</h2>
+
+<p class="govuk-body">
+  The OpenAPI spec from which this documentation is generated is <%= govuk_link_to 'available in YAML format', api_docs_spec_1_1_url %>.
+</p>
+
+<p class="govuk-body">
+  You can find a  <%= govuk_link_to 'summary of the field length limits', '#field-lengths' %> at the foot of this page.
+</p>
+
+<h3 class="govuk-heading-m">Environments</h3>
+
+<p class="govuk-body">
+  We have a production environment and a sandbox environment. When version 1.1 is launched, these environments will automatically upgrade to the latest minor version.
```

Comment:
instead explain strategy for Sandbox which is to specify the version first before releasing